src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoMonthGenitiveNames.cs

```diff
@@ -11,59 +12,13 @@ namespace System.Globalization.Tests
 {
     public class DateTimeFormatInfoMonthGenitiveNames
     {
-        // PosTest1: Call MonthGenitiveNames getter method should return correct value for InvariantInfo
-        [Fact]
-        public void TestGetter()
+        public static IEnumerable<object[]> MonthGenitiveNames_TestData()
         {
-            VerificationHelper(DateTimeFormatInfo.InvariantInfo,
-                    new string[] {
-                    "January",
-                    "February",
-                    "March",
-                    "April",
-                    "May",
-                    "June",
-                    "July",
-                    "August",
-                    "September",
-                    "October",
-                    "November",
-                    "December",
-                    ""
-                    },
-                    false);
-        }
-
-        // PosTest2: Call MonthGenitiveNames setter method should return correct value
-        [Fact]
-        public void TestSetter()
-        {
-            VerificationHelper(new DateTimeFormatInfo(),
-                    new string[] {
-                    "1",
-                    "2",
-                    "3",
-                    "4",
-                    "5",
-                    "6",
-                    "7",
-                    "8",
-                    "9",
-                    "10",
-                    "11",
-                    "12",
-                    ""
-                    },
-                    true);
-        }
-
-        // PosTest3: Call MonthGenitiveNames getter method should return correct value for ru-ru culture
-        [Fact]
-        public void TestRuCulture()
-        {
-            DateTimeFormatInfo info = new CultureInfo("ru-RU").DateTimeFormat;
-
-            string[] expected = new string[] {
+            yield return new object[] { DateTimeFormatInfo.InvariantInfo, new string[] { "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December", "" } };
+            yield return new object[] 
+            {
+                new CultureInfo("ru-RU").DateTimeFormat, new string[]
```

Comment:
I'd also chop it between the elements here, so it's more visible that the CultureInfo and the string array are different; but at least now the indentation level pops :)